### DISCUSSION TITLE
Understanding compare output

### DISCUSSION THREAD

**Discussion by athirar84** [ImageMagick/ImageMagick#5384] [Help]
$ magick compare -metric RMSE -subimage-search logo.png wizard.jpg similarity.tif
85.05 (0.00129778) @ 353,157 

Could someone please help to understand what is 0.00129778 in compare output ?

**Comment by fmw42** [comment]
It is the rmse score in the range 0 to 1 rather than the 85 which is in the range 0 to quantumrange (typically 16-bits or 65535 for the default build). So if you multiply the value in parenthesis by 100 you get 0.129778 % rmse error.

**Reply by athirar84** [reply]
Thank you!
